src/main/java/io/jenkins/plugins/luxair/ImageTagParameterDefinition.java

```diff
@@ -122,7 +130,7 @@ private String getDefaultOrEmptyCredentialId(String registry, String credentialI
             password = credential.getPassword().getPlainText();
         }
 
-        ResultContainer<List<String>> resultContainer = ImageTag.getTags(image, registry, filter, user, password, tagOrder);
+        ResultContainer<List<String>> resultContainer = ImageTag.getTags(image, registry, filter, user, password, tagOrder, verifySsl);
```

Comment:
```suggestion
        ResultContainer<List<String>> resultContainer = ImageTag.getTags(image, registry, filter, user, password, tagOrder, getVerifySsl());
```

With this change VerifySsl is never null :grinning: and the UniRest config always receives `true` or `false`

(note: I will do a similar change in a new PR for `tagOrder`, which has the same problem)